integration_test/ops_agent_test/main_test.go

```diff
@@ -4576,7 +4576,7 @@ traces:
 
 		// Have to wait for startup feature tracking metrics to be sent
 		// before we tear down the service.
-		time.Sleep(20 * time.Second)
+		time.Sleep(2 * time.Minute)
```

Comment:
Could we do the same update in the `TestOTLPMetricsGCM` test ?

[1] https://github.com/GoogleCloudPlatform/ops-agent/blob/master/integration_test/ops_agent_test/main_test.go#L4475

Reply:
Done